src/nodes.coffee

```diff
@@ -3097,12 +3211,13 @@ exports.If = class If extends Base
 
 UTILITIES =
   modulo: -> 'function(a, b) { return (+a % (b = +b) + b) % b; }'
-
+  objectWithoutKeys: -> 'function(obj, props) { return Object.keys(obj).reduce(function(a,c) { return (![].includes.call(props, c)) && (a[c] = obj[c]), a; }, {}); }'
```

Comment:
Hmm. Maybe we still have `hasProp`? I know I removed `indexOf`, in favor of native `[].indexOf`. All I meant was that you should rewrite your function to be independent, to not depend on other helpers that may or may not exist.